onnx_xla/onnx_xla_interface.cc

```diff
@@ -21,6 +24,13 @@ struct OnnxXlaBackendID {
   int device_id{0};
 };
 
+struct EventControl {
+  EventControl() : signalled(false) {}
+  bool signalled;
```

Comment:
This should be either `volatile bool`, or `std::atomic<bool>`

Reply:
Changed to volatile bool